sr_demos/scripts/sr_demos/demo.py

```diff
@@ -225,24 +230,27 @@ def _has_tactiles(self):
                 tactiles.append(tactile)
         return tactiles
 
-    def _get_joint_states_for_robot(self, joint_states_config_yaml):
+    def _get_joint_states_for_robot(self,
+                                    joint_states_config_yaml: Dict[str, Dict[str, int]]) -> Dict[str, Dict[str, int]]:
         '''
             Returns a dictionary of joint states with correct prefix
             and only joints that are present in the hand type
-            Input: joint_states_config_yaml: dictionary of joint states
-            Output: joint_states: dictionary of joint states
+            @param  joint_states_config_yaml: dictionary of joint states
+            @return joint_states: dictionary of joint states
```

Comment:
I don't think the return should have a named parameter after it
```suggestion
            @return: dictionary of joint states
```